core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -115,7 +125,21 @@ public int getValue() {
      */
     UnderFileSystem get(String path, Object ufsConf, Configuration configuration) {
       UnderFileSystem cachedFs = null;
-      Key key = new Key(new AlluxioURI(path));
+      PermissionStatus ps = PermissionStatus.defaults();
+      try {
+        // TODO(chaomin): consider adding a JVM-level constant to distinguish between Alluxio server
```

Comment:
@calvinjia @apc999 @gpang is there an idiomatic way to determine if the current process is Alluxio master vs. Alluxio worker vs. Alluxio client?